docs/changes/cosmology/12100.other.rst

```diff
@@ -0,0 +1 @@
+Developer notes in the code and docs are moved to a dedicated page.
```

Comment:
I think this fragment name violates the towncrier rules. See #11945

Reply:
Ah. Thanks. I'll push a fix.
What rule specifically?